circle.yml

```diff
@@ -8,12 +8,26 @@ machine:
     PATH: $HOME/.cabal/bin/:$PATH
 
 
+# checkout:
+#
+  # override:
+  #   - git clone --depth=2 --branch=$CIRCLE_BRANCH https://github.com/Originate/git-town.git /home/ubuntu/.go_workspace/src/github.com/Originate/gt
+  #   - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt
+  #   - git checkout $CIRCLE_SHA1
+
+
 dependencies:
 
   # Cache ShellCheck for subsequent builds
   cache_directories:
     - "~/.cabal"
 
+  override:
+    - mkdir -p /home/ubuntu/.go_workspace/src/github.com/Originate
+    - cp -r /home/ubuntu/git-town /home/ubuntu/.go_workspace/src/github.com/Originate/gt
```

Comment:
CircleCI clones the repo into `/home/ubuntu/git-town`, Go expects source code to exist in `$GOPATH/src/github.com/Originate/gt`. In a pure Go codebase it works, but here we mix Ruby in, so I am copying over the code into the correct folder manually in order to compile it there.